frontend/common/Masthead.tsx

```diff
@@ -74,6 +75,10 @@ function isEdaServer(): boolean {
   return process.env.UI_MODE === 'EDA';
 }
 
+function isHubServer(): boolean {
+  return process.env.UI_MODE === 'HUB';
```

Comment:
I'd like to move away from accessing the env variables in the components. I'd suggest passing in the api URLs via a prop. Each app can pass this in where the Masthead is rendered in AwxMain, HubMain, etc

Reply:
Im working on abstracting the masthead and making it composable.
That way we can have an AwxMasthead that contains a menu that has a logout that is AWX specific. etc...

Reply:
@keithjgrant Thanks for your review, as James mentioned there is a long term goal to refactor the masthead to achieve what you described.